politeiawww/backend.go

```diff
@@ -1555,6 +1555,20 @@ func (b *backend) ProcessProposalDetails(propDetails www.ProposalsDetails, isUse
 			Signature:        cachedProposal.Signature,
 			CensorshipRecord: cachedProposal.CensorshipRecord,
 		}
+
+		stringUserID := b.userPubkeys[cachedProposal.PublicKey]
+
+		var userID uint64
+		if stringUserID != "" {
+			userID, err = strconv.ParseUint(stringUserID, 10, 64)
+			if err != nil {
+				return nil, err
+			}
+		}
+
+		if user != nil && user.ID == userID {
```

Comment:
This is a nitpick but the above logic should all go within an `if user != nil` block because it doesn't apply otherwise. In other words, I would do this:

    if user != nil {
        // ... find userID ...
        
        if user.ID == userID {
            reply.Proposal.Name = cachedProposal.Name
        }
    }